File: foundations/py/3/counter.py
print('Please enter the number of coins:')
quarters = int(input('# of quarters:'))
dimes = int(input('# of dimes:'))
nickels = int(input('# of nickels:'))
pennies = int(input('# of pennies:'))

cents = quarters * 25 + dimes * 10 + nickels * 5 + pennies

print('The total is', (cents // 100), 'dollars and', (cents % 100), 'cents')

File: foundations/cpp/4/counter.cpp
#include <iostream>
using namespace std;

int main()
{
	int quarters, dimes, nickels, pennies;
	cout << "Please enter the number of coins:" << endl;
	cout << "# of quarters: ";
	cin >> quarters;
	cout << "# of dimes: ";
	cin >> dimes;
	cout << "# of nickels: ";
	cin >> nickels;
	cout << "# of pennies: ";
	cin >> pennies;
	int cents = quarters * 25 + dimes * 10 + nickels * 5 + pennies;
	cout << "The total is " << cents / 100 << " dollars and " << cents % 100 << " cents" << endl;
	return 0;
}